cmake/modules/dts.cmake

```diff
@@ -158,15 +161,10 @@ if(DTC_OVERLAY_FILE)
     )
 endif()
 
-set(i 0)
 foreach(dts_file ${dts_files})
-  if(i EQUAL 0)
-    message(STATUS "Found BOARD.dts: ${dts_file}")
-  else()
+  if(dts_file MATCHES ".*[.]overlay$")
```

Comment:
What if someone provides a file with a different file extension? What if someone on windows provides a file with the extension in CAPS LOCK mode?